src/Apps/Artwork/routes.tsx

```diff
@@ -11,11 +12,22 @@ export const routes = [
     path: "/artwork/:artworkID/(confirm-bid)?",
     Component: ArtworkApp,
     query: graphql`
-      query routes_ArtworkQuery($artworkID: String!) {
+      query routes_ArtworkQuery(
+        $artworkID: String!
+        $enablePricingContext: Boolean!
+      ) {
         artwork(id: $artworkID) {
           ...ArtworkApp_artwork
         }
       }
     `,
+    prepareVariables: (params, { context }) => ({
+      ...params,
+      enablePricingContext: get(
+        context,
+        ctx => ctx.user.lab_features.includes("Pricing Context"),
```

Comment:
This ends up being so clean!

Is it possible that `user.lab_features` will return `nil` here? Or `ctx.user`?